### PR TITLE
Andyderek patch stylelint

### PR DESCRIPTION
**Pull request by codecakes**
This change is [Reviewable](https://reviewable.io/reviews/xcov19/mycovidconnect/78)

### CODE REVIEW

File: .github/workflows/csslint.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name: Stylesheet Lint
```
Comment (codecakes) [inline .github/workflows/csslint.yml:1]: @andyderek see #73 esp https://github.com/Xcov19/mycovidconnect/pull/73/checks?check_run_id=1172342856 for why its failing.

See https://github.com/Xcov19/mycovidconnect/blob/1427edb544c2505e90186f7212909382e8023611/.github/workflows/csslint.yml for how ive added actions

**Review by codecakes — COMMENTED** [review]
See https://github.com/Xcov19/mycovidconnect/blob/1427edb544c2505e90186f7212909382e8023611/.github/workflows/csslint.yml for how ive added actions

File: .github/workflows/csslint.yml (hunk 1 above)
Comment (codecakes) [reply .github/workflows/csslint.yml:1]: @andyderek if the yml file is not added like this, it won't run in the pipeline. See the pipeline checks below

Comment (andyderek) [reply .github/workflows/csslint.yml:1]: I was just trying to see why it cannot find the css files.  That’s what it’s saying in the errors.

Comment (codecakes) [reply .github/workflows/csslint.yml:1]: you need to check how ive added in the workflow to make it work in this PR; 
the css patterns arent matching

File: public/index.html [hunk 2]
```diff
@@ -29,5 +29,18 @@
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
+     <!--Start of Tawk.to Script-->
```
Comment (codecakes) [inline public/index.html:32]: why is this script in this PR?

**Review by codecakes — COMMENTED** [review]
tawk.to script is already merged in another PR. pls remove

**Pull request closed without merging**